src/core/mac/mac.cpp

```diff
@@ -1343,6 +1344,7 @@ void Mac::ReceiveDoneTask(Frame *aFrame, ThreadError aError)
         {
             for (Receiver *receiver = mReceiveHead; receiver; receiver = receiver->mNext)
             {
+                otDumpDebgMac("RX", aFrame->GetHeader(), aFrame->GetLength());
```

Comment:
In this case, I was wondering if the dump should be outside the for-loop.  Inside is useful if the handlers modify the frame buffer.